apps/site/pages/docs/building-sections/overrides/override-component.mdx

```diff
@@ -0,0 +1,244 @@
+                  "type": "object",
+                  "properties": {
+                    "text": {
+                      "title": "Link Text",
+                      "type": "string",
+                      "default": "I don't know my Postal Code"
+                    },
+                    "to": { "title": "URL", "type": "string" }
+                  }
+                },
+                "shippingOptionsTableTitle": {
+                  "title": "Shipping Options Table Header",
+                  "type": "string"
+                }
+              }
+            },
+            "productDescription": {
+              "title": "Product Description",
+              "type": "object",
+              "properties": {
+                "initiallyExpanded": {
+                  "type": "string",
+                  "title": "Initially Expanded?",
+                  "enumNames": ["First", "All", "None"],
+                  "enum": ["first", "all", "none"]
+                },
+                "displayDescription": {
+                  "title": "Should display description?",
+                  "type": "boolean",
+                  "default": true
+                },
+                "title": {
+                  "title": "Description section title",
+                  "type": "string",
+                  "default": "Description"
+                }
+              }
+            }
+          }
+        }
+      }
+  ]
+```
+
+This schema Adds the new `Custom Buy Button` section to the Headless CMS.
+
+5. Open a new terminal window and log in to your vtex account by running `vtex login {accountName}`.
+
+6. run the following command to sync your changes with the Headless CMS:
+
+```bash
+yarn cms-sync
+```
+7. Access the VTEX Admin and proceed to **Storefront > Headless CMS**.
+
+8. Click on the page you desire to add the new section.
+
+9. Click the add button (`+`) and add the new section and complete values from the sections fields.
+
+10. Click `Save`.
+
+> ⚠️ Remember: props changed via Headless CMS overlap the changes made through [overrides](LINK).
+  For example, if you change a prop through override and also change it using
+  Headless CMS, the final prop's value will be the one added using CMS.
+
+### Step 4 - Visualizing the component on the page locally
+
+To see your changes locally you need to set the Headless CMS preview to the development mode. 
+Refer to the [How to preview Headless CMS changes in development mode](/docs/how-to-preview-headless-cms-changes-in-development-mode) guide for more information.
+
+Once you have set the preview for development mode, you should see your new Buy Button in a Product Details Page (PDP).
+
+### Step 5 - Publishing your changes
+
+If your changes are working in the development mode and you want to publish them, follow the steps below:
+
+1. Go to the Headless CMS in the VTEX Admin.
+2. Click on a PDP page.
+3. Click `Publish` to make the new button available on the production store.
```

Comment:
suggestion: Add a new section below this one to refer [Use Cases Examples](https://faststore-site-git-docs-override-version-2-faststore.vercel.app/docs/building-sections/overrides/use-cases/overrides-use-cases-overview), so people can have more linked doc to look for.

Reply:
suggestion: Is it possible to change the order this page appear in the list?

I think is better to show in the following order: 
- Overrides use cases
- Add other icons option to the Alert component
- Add an image to your Alert component


<img width="275" alt="image" src="https://github.com/vtex/faststore/assets/3356699/75169fd5-c532-47fb-b4d6-96b92fff341f">